ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/CompletedExceptionRequestModal.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Button, Modal } from '@patternfly/react-core';
+import { VulnerabilityException } from 'services/VulnerabilityExceptionService';
+
+export type CompletedExceptionRequestModalProps = {
+    exceptionRequest: VulnerabilityException;
+    onClose: () => void;
+};
+
+function CompletedExceptionRequestModal({
+    exceptionRequest,
+    onClose,
+}: CompletedExceptionRequestModalProps) {
+    const title = 'TODO';
+
+    return (
+        <Modal
+            onClose={onClose}
+            title={title}
+            isOpen
+            variant="medium"
+            actions={[
+                <Button key="confirm" variant="primary" onClick={onClose}>
+                    Done
```

Comment:
What do you think about **Close** which is a verb compared to **Done** which is a adjective?

Reply:
The extra step is a puzzle. TODO for title is evidence of that.

The following interaction seems more direct and usual in **Request deferral** modal:
* No close button at upper right, because redundant with Cancel, and not easy to disable.
* Set `isDisabled` and `isLoading` props of **Submit request** button.
* Set `isDisabled` prop of **Cancel** button.
* If request fails, display error in modal, else close modal.

Reply:
Bravo, this contribution does add the props!

Reply:
Agreed, I'll update to Close.

The reason for this extra step is to show the user a link to the successfully submitted request once a response has returned, with the intention that the user will share the link with an approver on their team.
![image](https://github.com/stackrox/stackrox/assets/1292638/dc6ecf02-819a-4ab3-8b8b-00848977f9dc)